README.md

```diff
@@ -198,9 +198,7 @@ the GitHub project's master branch.
 
 ## Building Dagger
 
-Dagger is built with [`bazel`]. The tests can be run with `bazel test //...`.
-`util/install-local-snapshot.sh` will build all of the Dagger libraries and
-install a copy in your local maven repository with the version `LOCAL-SNAPSHOT`.
+See [Building Dagger][builder-dagger] in the contributing guidelines.
```

Comment:
You can remove the [builder-dagger] part and rename the link below to "[Building Dagger]" and it'll work (and be slightly cleaner)